samples/drivers/led_pwm/boards/rpi_pico.overlay

```diff
@@ -0,0 +1,7 @@
+&pwm_led0 {
+	status = "okay";
+};
+
+&pwm {
+	status = "okay";
+};
```

Comment:
can you please explain this better? Why can't the sample run on rpi pico?

Reply:
The PWM peripheral is clocked by `&system_clk` (will fix this in the driver), which by default runs at 125MHz on the `rpi_pico` board. This can be divided down by ~256 (255 15/16 in fact) resulting in a frequency of ~490 KHz as the lowest frequency that can be achieved by the pwm peripheral, resulting in a period of ~2 us. The pwm_led example tries to blink the onboard LED using the pwm peripheral with a period of 0.1 s. The part of the sample that does work is fading the LED since this requires far shorter periods of the PWM peripheral.